app/models.py

```diff
@@ -4,7 +4,8 @@
 SQLAlchemy models for the app
 '''
 
-from app import ORG_TYPES, VALID_SKILL_LEVELS, QUESTIONS_BY_ID, LEVELS
+from app import ORG_TYPES, VALID_SKILL_LEVELS, QUESTIONS_BY_ID, LEVELS, \
+                QUESTIONNAIRES
```

Comment:
Ah good idea, I'm not sure what PEP 8 says but I like the parenthesis better regardless.